libvast/src/store.cpp

```diff
@@ -177,30 +314,30 @@ system::store_builder_actor::behavior_type default_active_store(
   self->state.store = std::move(store);
   self->state.path = std::move(path);
   self->state.store_type = std::move(store_type);
+  self->set_down_handler([self](const caf::down_msg& down_msg) {
+    for (const auto& [query_id, state] : self->state.running_extractions) {
```

Comment:
Seems to be duplicated twice. How about a free function in anon namespace.

[self](const caf::down_msg& down_msg)
{
    handle_down_msg(self, down_msg);
}

Reply:
Nice idea, thanks for that! Applied.